src/main/java/org/testeditor/web/xtext/index/XtextIndexHelloWorldApplication.xtend

```diff
@@ -0,0 +1,31 @@
+package org.testeditor.web.xtext.index
+
+import io.dropwizard.Application
+import io.dropwizard.setup.Bootstrap
+import io.dropwizard.setup.Environment
+import org.testeditor.web.xtext.index.resources.XtextIndexHelloWorldResource
+import org.testeditor.web.xtext.index.health.XtextIndexTemplateHealthCheck
+
+class XtextIndexHelloWorldApplication extends Application<XtextIndexHelloWorldConfiguration> {
+    def static main(String[] args) throws Exception {
+        new XtextIndexHelloWorldApplication().run(args);
```

Comment:
() ; are relics from the old java world!

Reply:
semicolon is redundant, braces of constructor call are not.